src/main/java/org/opentripplanner/middleware/utils/NotificationUtils.java

```diff
@@ -327,7 +329,7 @@ public static int getPushInfo(String toUser) {
         try {
             Map<String, String> headers = Map.of("Accept", "application/json");
             var httpResponse = HttpUtils.httpRequestRawResponse(
-                URI.create(PUSH_API_URL + "/devices/get?api_key=" + PUSH_API_KEY + "&user=" + toUser),
+                URI.create(getPushDevicesUrl(PUSH_API_URL + "/devices/get?api_key=" + PUSH_API_KEY + "&user=", toUser)),
```

Comment:
Nit. This might now benefit from a String.format?

Reply:
Good idea, done in https://github.com/ibi-group/otp-middleware/pull/202/commits/0e8ba243917c8df7c997c4825efa222259f95153.